web/client/plugins/__tests__/AddGroup-test.jsx

```diff
@@ -73,8 +73,6 @@ describe('AddGroup Plugin', () => {
         const btn = document.getElementsByTagName('button')[1];
         ReactTestUtils.Simulate.click(btn);
         expect(store.getState().layers.groups.length).toBe(2);
-        expect(store.getState().layers.groups[1].name).toBe('newgroup');
```

Comment:
Hi, if you are removing bcause the id is generated each time maybe you can test the length of it.
like if using uuid is 32 chars